src/chunk.c

```diff
@@ -2392,11 +2392,17 @@ ts_chunk_drop_process_materialization(Oid hypertable_relid,
 	ts_cache_release(hcache);
 }
 
+/* Continuous agg materialization hypertables can be dropped
+ * only if a user explicitly specifies the table name
+ * see discussion related to drop_chunks API:
+ * https://github.com/timescale/timescaledb/pull/1648
+ * API needs to be deprecated. this is the short term fix until then.
```

Comment:
I am not 100% convinced this API needs deprecation, as discussed before. Let's remove this part of the comment for now and continue the discussion.